### PR TITLE
Shooter

### PR DESCRIPTION
**Pull request by Sahaj8**

### CODE REVIEW

**Review by satu0king — CHANGES_REQUESTED** [review]
Try and learn how to work with branches. Then you wouldn't need to delete index.html and create shoot.html
Also your PR should have just 1-2 commits after master.

**Comment by Kartik-Sama** [comment]
While deceasing velocity shouldn't speed approach 0, instead of again increasing in negative direction ?

File: canvas.js [hunk 1]
```diff
@@ -206,7 +206,7 @@ canvas.setupFunction = function() {
 
 // Called when key is pressed
 canvas.keyDownCallback = function (e) {
-    console.log("Dummy KeyDownCallback - Override canvas.KeyDownCallback");
```
Comment (satu0king) [inline canvas.js:209]: Why is there a change here??

**Review by satu0king — CHANGES_REQUESTED** [review]
I think there is a mistake in the PR, I cannot see the shooter code .. Maybe create a new PR ?

File: canvas.js (hunk 1 above)
Comment (Sahaj8) [reply canvas.js:209]: I will change it and send a new pr.

**Comment by satu0king** [comment]
@Sahaj8 should we close this PR?

**Pull request closed without merging**